src/test/resources/fakeExternalMovementsData.json

```diff
@@ -0,0 +1,5002 @@
+[
```

Comment:
Do you need the full data set in the tests? Could this be cut down, or could the main data set be used instead?

Reply:
The test data set has been truncated now.